frontend/website/bsconfig.json

```diff
@@ -10,7 +10,8 @@
     "bs-css"
   ],
   "warnings": {
-    "error": "+101"
+    "error": "+101",
+    "number": "-40-44+6+7+27+32..39+45"
```

Comment:
are these standard convention somewhere? Looking at these (https://caml.inria.fr/pub/docs/manual-ocaml/comp.html#sec288) and curious turn certain warnings on vs off

Reply:
We have standard set for the protocol build, in `src/dune`.

Reply:
those are pretty different: 
```
(env
  (_ (flags (:standard -short-paths -w @a-4-29-40-41-42-44-45-48-58-59-60))))
```

These align more with bs defaults (https://reasonml.chat/t/what-warnings-errors-do-you-always-enable-or-disable/392). Anyway, just curious, not blocking the merge at all

Reply:
The number here is bs-platform default with -44 added to remove the error about local open shadowing things. We can re-enable this error if we want.

Reply:
@o1pranay I could also change it to something like `@a-40-42-44`, which is the minimum number we would need to disable in order to get it to compile without warning currently.

Reply:
@Schmavery no need, was just curious - feel free to merge!